sw/airborne/math/pprz_algebra_float.c

```diff
@@ -609,38 +713,38 @@ float float_mat_det_4d(float m[16]);
 
```

Comment:
The lines above this shouldn't be needed, can you remove them plz :) less more is more better...

Also, if you are super generous, can you add function descriptors for some more of these functions (very optional)?

Reply:
Without this I get a warning for a missing prototype...